tests/foreman/ui/test_activationkey.py

```diff
@@ -980,8 +981,8 @@ def test_positive_add_host(self):
             with VirtualMachine(distro=self.vm_distro) as vm:
                 vm.install_katello_ca()
                 vm.register_contenthost(self.organization.label, key_name)
-                name = self.activationkey.fetch_associated_content_host(
-                    key_name)
+                name = self.activationkey.fetch_associated_content_hosts(
+                    key_name)[0]
```

Comment:
Frankly both errors are pretty straightforward for me, but you're right, will update, thanks for suggestion.